modules/react/select/stories/stories_VisualTesting.tsx

```diff
@@ -65,21 +61,19 @@ export const SelectStates = () => {
         {props => (
           <FormField>
             <FormField.Label>Contact</FormField.Label>
-            <Select model={model}>
+            <Select items={options} nonInteractiveIds={disabledItems}>
               <FormField.Input as={Select.Input} {...props} id="contact-select" />
               <Select.Popper>
                 <Select.Card maxHeight="200px">
-                  {model.state.items.length > 0 && (
-                    <Select.List>
-                      {item => {
-                        return (
-                          <Select.Item aria-disabled={item.disabled ? item.disabled : undefined}>
-                            {item.id}
-                          </Select.Item>
-                        );
-                      }}
-                    </Select.List>
-                  )}
+                  <Select.List>
+                    {item => {
+                      return (
+                        <Select.Item aria-disabled={item.disabled ? item.disabled : undefined}>
```

Comment:
Can line 52 look like this: `{label: '', value: undefined},`? So you will not need this ternary

Reply:
Yup, updated